tests/collections/common/tokenizers/text_to_speech/test_tts_tokenizers.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import ctypes.util
+import os
+
+import pytest
+
+from nemo.collections.common.tokenizers.text_to_speech.tts_tokenizers import PhonemizerTokenizer
+
+
+class TestTTSTokenizers:
```

Comment:
Can you add an oov test here? What happens if I pass in emoji to `en-us` or if I pass in Korean to `de`

Reply:
If any character not in the chars list is present, it removes that character from the text. And it prints a message like:

Text: [ɛnˈvidiə🙂 ˈnimoʊ 🧐] contains unknown char/phoneme: [🧐].Original text: [NVIDIA🙂 NeMo 🧐]. Symbol will be skipped.

So the user is informed that they need to add new characters to their alphabet.

I would expect the unknown characters to be replaced with the "oov" token that is provided in every Tokenizer constructor. However currently that "oov" token is never used anywhere.